administration/src/cards/PdfFactory.ts

```diff
@@ -29,17 +46,20 @@ const fillContentAreas = async (
   deepLink: string,
   region?: Region
 ): Promise<void> => {
-  const helveticaFont = await doc.embedFont(StandardFonts.Helvetica)
+  const font = await doc.embedFont(StandardFonts.Helvetica)
   pdfConfig.elements?.dynamicActivationQrCodes.forEach(configOptions =>
     pdfQrCodeElement(configOptions, { page: templatePage, qrCode: dynamicCode })
   )
 
+  const fontBold = pdfConfig.customBoldFont
+    ? await loadCustomFontWithFallback(pdfConfig.customBoldFont, doc, StandardFonts.HelveticaBold)
+    : await doc.embedFont(StandardFonts.HelveticaBold)
```

Comment:
:question: Do I understand this correctly that the normal font is the same for koblenz and only for the bold font they want some custom font?

Reply:
Yes, we do only use bold fonts in koblenz pdf